### DISCUSSION TITLE
Crashpad_handler doesn't generate minidump if app is not started by admin

### DISCUSSION THREAD

**Discussion by martirosyanH** [getsentry/sentry-docs#7715] [Q&A]
Hi,
 I have integrated sentry-native to my application. everything seems to work when I start my exe with admin.
 But when I do regular start of my application it generates all the folder strcture, creates reports folder, but mini dump file is not generated.
 
 I tried to set sentry_options_set_database_pathw to some other directory, where admin acess is not needed, but same issue persist.
 
 Any help would be appreciated.

Thank you.

**Comment by kahest** [comment]
Closing this discussion in favor of https://github.com/getsentry/sentry-native/discussions/878
